kubernetes-elasticsearch-cluster/es-data-stateful.yaml

```diff
@@ -50,14 +52,16 @@ spec:
         - name: HTTP_ENABLE
           value: "false"
         - name: ES_JAVA_OPTS
-          value: -Xms1g -Xmx1g
+          value: -Xms7g -Xmx7g
```

Comment:
I assume this doesn't work with default machine type of n1-standard-1. Please revert everything in this file except for lines 23-24.